src/Components/Facility/CoverImageEditModal.tsx

```diff
@@ -129,125 +178,275 @@ const CoverImageEditModal = ({
+            </div>
+            {/* buttons for mobile screens */}
+            <div className="flex justify-evenly m-4 sm:hidden ">
+              <div>
+                {!previewImage ? (
+                  <ButtonV2 onClick={handleSwitchCamera} className="m-2">
+                    Switch
+                  </ButtonV2>
+                ) : (
+                  <></>
+                )}
+              </div>
+              <div>
+                {!previewImage ? (
+                  <>
+                    <div>
+                      <ButtonV2
+                        onClick={() => {
+                          captureImage();
+                        }}
+                        className="m-2"
+                      >
+                        Capture
+                      </ButtonV2>
+                    </div>
+                  </>
+                ) : (
+                  <>
+                    <div className="flex space-x-2">
+                      <ButtonV2
+                        onClick={() => {
+                          setPreviewImage(null);
+                        }}
+                        className="m-2"
+                      >
+                        Retake
+                      </ButtonV2>
+                      <ButtonV2 onClick={handleUpload} className="m-2">
+                        {isCaptureImgBeingUploaded ? (
+                          <CareIcon className="care-l-spinner text-lg animate-spin" />
+                        ) : (
+                          <></>
+                        )}{" "}
+                        Submit
+                      </ButtonV2>
+                    </div>
+                  </>
+                )}
+              </div>
+              <div className="sm:flex-1">
+                <ButtonV2
+                  variant="secondary"
+                  onClick={() => {
+                    setPreviewImage(null);
+                    setIsCameraOpen(false);
+                    webRef.current.stopCamera();
+                  }}
+                  className="m-2"
+                >
+                  Close
+                </ButtonV2>
+              </div>
+            </div>
+            {/* buttons for laptop screens */}
+            <div className={`${isLaptopScreen ? " " : " hidden "}`}>
+              <div className="flex m-4 lg:hidden">
+                <ButtonV2 onClick={handleSwitchCamera}>
+                  <CareIcon className="care-l-camera-change text-lg" /> Switch
+                  Camera
+                </ButtonV2>
+              </div>
+
+              <div className="flex justify-end  p-4 gap-2">
+                <div>
+                  {!previewImage ? (
+                    <>
+                      <div>
+                        <ButtonV2
+                          onClick={() => {
+                            captureImage();
+                          }}
+                        >
+                          <CareIcon className="care-l-capture text-lg" />{" "}
```

Comment:
`ButtonV2` applies spacing by default with `gap-2`
```suggestion
                          <CareIcon className="care-l-capture text-lg" />
```